src/proxy-ui-api/frontend/src/views/AppLogin.vue

```diff
@@ -265,16 +266,21 @@ export default (
         });
     },
 
-    async fetchCurrentSecurityServer() {
+    fetchCurrentSecurityServer() {
       this.$store.dispatch('fetchCurrentSecurityServer').catch((error) => {
         this.$store.dispatch('showError', error);
       });
     },
-    async fetchSecurityServerVersion() {
+    fetchSecurityServerVersion() {
       this.$store
         .dispatch('fetchSecurityServerVersion')
         .catch((error) => this.$store.dispatch('showError', error));
     },
+    fetchSecurityServerNodeType() {
+      this.$store.dispatch('fetchSecurityServerNodeType').catch((error) => {
```

Comment:
```suggestion
      return this.$store.dispatch('fetchSecurityServerNodeType').catch((error) => {
```

Reply:
Refactored to an async function with blocking vuex dispatch